src/UIKit/UITypes.cs

```diff
@@ -201,13 +201,11 @@ public UIFloatRange (nfloat minimum, nfloat maximum)
 			}
 		}
 
-		[DllImport (Constants.UIKitLibrary)]
-		static extern bool UIFloatRangeIsEqualToRange (UIFloatRange range, UIFloatRange otherRange);
+		// Got replaced by a macro in iOS 13 (Xcode 11)...
+		// [DllImport (Constants.UIKitLibrary)]
+		// static extern bool UIFloatRangeIsEqualToRange (UIFloatRange range, UIFloatRange otherRange);
 
-		public bool Equals (UIFloatRange other)
-		{
-			return UIFloatRangeIsEqualToRange (this, other);
-		}
+		public bool Equals (UIFloatRange other) => this.Minimum == other.Minimum && this.Maximum == other.Maximum;
```

Comment:
This is absolutely bizarre because the documentation says that it uses an epsilon for the comparison. The swift equivalent does exactly this. Comparing floating point numbers for equality is generally a Very Bad Idea.

Reply:
So bad that it breaks our tests...

Reply:
-> feedback

Reply:
Feedback sent, will add to our tracker. https://feedbackassistant.apple.com/feedback/6629947